### PR TITLE
New RCON commands feature

### PR DESCRIPTION
**Pull request by shotah**
Adding the ability to run RCON commands at server start, client connect, and client disconnect.

### CODE REVIEW

**Comment by shotah** [comment]
@itzg Hopefully this gives additional flexibility to the image.

File: files/rconcmds/rcon-cmds-daemon.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+# needed for the clients connected function residing in autopause
+# shellcheck source=/autopause/autopause-fcns.sh
+. /autopause/autopause-fcns.sh
+
+# shellcheck source=start-utils
+. ${SCRIPTS:-/}start-utils
+
+run_comand(){
+  rcon_cmd=$(echo "$1" | tr -d '"')
```
[the remark this answers is not in the document]
Comment (shotah) [reply files/rconcmds/rcon-cmds-daemon.sh:15]: This was to trim anything off. Can be removed, I think I added this when going through string hell.

**Comment by shotah** [comment]
@itzg Made requested changes.

File: files/rconcmds/rcon-cmds-daemon.sh [hunk 2]
```diff
@@ -11,11 +12,11 @@
 # shellcheck source=start-utils
 . ${SCRIPTS:-/}start-utils
 
-run_comand(){
-  rcon_cmd=$(echo "$1" | tr -d '"')
-  logAutopause "RCON: running - $rcon_cmd"
+run_command(){
+  rcon_cmd=$1
```
Comment (itzg) [inline files/rconcmds/rcon-cmds-daemon.sh:16]: I'm surprised bash didn't require this to be quoted as
```suggestion
  rcon_cmd="$1"
```

**Review by itzg — COMMENTED** [review]
Just the one syntax question

**Review by itzg — APPROVED** [review]
Thanks!

**Pull request merged**